schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+CREATE TABLE ANIMALS (
+   ID INT PRIMARY KEY NOT NULL,
+   NAME TEXT NOT NULL,
```

Comment:
- [ ] Setting limits on `VARCHAR` columns in a database ensures data integrity, improves performance, and enhances security by preventing overly long or inappropriate data. Kindly add a limit to the name field 👍